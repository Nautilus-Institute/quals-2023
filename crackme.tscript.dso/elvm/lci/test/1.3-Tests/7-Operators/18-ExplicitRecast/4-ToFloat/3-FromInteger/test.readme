This test checks that the explicit recast operator correctly casts integer
values to floating point decimal types.
