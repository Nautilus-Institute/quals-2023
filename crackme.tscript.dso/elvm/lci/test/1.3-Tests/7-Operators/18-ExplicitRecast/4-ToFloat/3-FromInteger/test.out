0.00
1.00
123456.00
-123456.00
